com.unity.sg2/Editor/GraphDeltaRegistry/Generation/Interpreter.cs

```diff
@@ -9,11 +9,17 @@ namespace UnityEditor.ShaderGraph.Generation
 {
     public static class Interpreter
     {
-        public static string GetFunctionCode(NodeHandler node, Registry registry)
+        public static string GetFunctionCode(NodeHandler node, Registry registry) // What is this used for?
```

Comment:
Just responding to the comment for context:

There's a collection of required and potentially useful pieces of data that are cached as a part of the preview management. Things like the compiled shader, material, the shader code, etc. This function caches the code local _only_ to the node, whereas the subsequent function `GetBlockCode` provides the code for all upstream nodes.

I would like us to have way for users to inspect the local code gen, as opening preview shaders or generated shaders can be full of noise if it's not necessary.

Reply:
Changed - removed comment. Thank you for the description.